app/javascript/controllers/alerts_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["list"];
+  connect() {
+    this.showAlert();
+  }
+
+  showAlert() {
```

Comment:
Is this showing the entire list of alerts? If so, can we change the method name to #showAlerts()?

Reply:
Yes, it is showing all the alerts, can change it, no problem.

Reply:
To be precise, it is showing the whole div that has all the alerts embedded in.